transaction-pool/src/tests/mod.rs

```diff
@@ -125,7 +125,7 @@ fn should_reject_if_above_count() {
 	let tx2 = b.tx().nonce(1).new();
 	let hash = *tx2.hash();
 	txq.import(tx1).unwrap();
-	assert_eq!(txq.import(tx2).unwrap_err().kind(), &error::ErrorKind::TooCheapToEnter(hash, "0".into()));
+	assert_eq!(txq.import(tx2).unwrap_err().kind(), &error::ErrorKind::TooCheapToEnter(hash, "0x0".into()));
```

Comment:
In other places I saw "0x" variant. How is this different? I think we should use the same style everywhere.

Reply:
`0x` is used to encode empty set of bytes, with numbers we never skip leading zeros, but representation of `0` is an exception (`0x0`). The string encoding is the same as in json-rpc, this particular change was required after bumping `ethereum-types`, where we changed `Display` and `Debug` implementations of uints to be more consistent (i.e. prefix with `0x` every time we display hex).